myhoard/restore_coordinator.py

```diff
@@ -40,6 +40,8 @@ class RestoreCoordinator(threading.Thread):
     ITERATION_SLEEP_SHORT = 0.2
     ITERATION_SLEEP_LONG = 10
 
+    MAX_BASEBACKUP_ERRORS = 4
```

Comment:
This could use a short comment on telling what kind of errors (do we ignore up to MAX_BASEBACKUP_ERRORS while retrying, or is this maximum number of retries?).

Reply:
Added comment.